js/generate.js

```diff
@@ -0,0 +1,23 @@
+import { randomNumber } from './util';
+import { getRandomArrayElement, NAMES, DESCRIPTIONS, MESSAGES } from './data';
```

Comment:
Тут не хватает расширения импортируемого файла
```suggestion
import { randomNumber } from './util.js';
import { getRandomArrayElement, NAMES, DESCRIPTIONS, MESSAGES } from './data.js';
```